jupyter_book/07-preprocessing-steps/preprocessing-steps.md

```diff
@@ -0,0 +1,17 @@
+# Preprocessing steps
+
+Before inputting the data into the satay pipeline, it is necessary to preprocess the data that comes from the sequencing company. 
+
+The pipeline understands single end sequencing data, in which the data from every digestion is mixed together. You dont need to analyze them separately. 
+
+## What we do if the sequencing data is paired end: 
+
+- Convert the data to single end by:
+    - Extracting the forward reads, which are the reads that contain the sequencing primer, as it is (harsh filtering) or allowing some mismatches in the sequencing primer, due to likely sequencing errors (gentle filtering). 
+    - We use a bash script for each type of filtering, if you would like to use it , please contact us by writing [here](mailto:L.M.InigoDeLaCruz@tudelft.nl).
+- Remove the sequence after the first restriction site for NiaIII and DpnII to avoid having chimeras sequences in our data, that have poor alignment.
```

Comment:
So these steps actually do not take place when running the pipeline

Reply:
No, they are the preprocessing steps , before running the pipeline , that is why I put that we use a bash script for it , and I said that in the beginning _Before inputting the data into the satay pipeline_

Reply:
This comment was meant as a follow-up on the part that says: 

> The pipeline understands single end sequencing data, in which the data from every digestion is mixed together.

Because the pipeline itself does not even recognize that the reads come from different digestions

Reply:
No, exactly, so it is OK to mix them , it saves more analysis steps .

Reply:
yes it is a not a requirement , it is an advantage I will say

Reply:
As in my previous comment, it does save analysis steps, but that shortcut can affect the results/increase the probability of misalignment of some reads. How big that problem is probably depends on what your initial sample looked like

Reply:
yes , that will be  a problem if we dont do the trimming of the downstream sequence after the first restriction site . So in order to decrease the probability of misalignment of some reads, we do this pre-processing step